openmetadata-ui/src/main/resources/ui/src/styles/components/react-awesome-query.less

```diff
@@ -208,3 +215,38 @@
   overflow: hidden;
   opacity: 1;
 }
+
+.rule-container .ant-btn-group {
+  visibility: hidden;
+}
+.rule-container + .rule-container .ant-btn-group {
+  visibility: visible;
+}
+
+.rule--field {
+  width: 180px;
+
+  .ant-select {
+    width: 100% !important;
```

Comment:
See if we can avoid the `!important` here.

Reply:
I am still figuring out a way to set width via config, but till then `!important` is the option I am left with